packages/openshift/result.py

```diff
@@ -3,9 +3,14 @@
 
 
 class Result(object):
-    def __init__(self, high_level_operation):
+    def __init__(self, high_level_operation, track_limit = None):
```

Comment:
For consistency, lets call this `tracking_limit` vs `track_limit`.

Reply:
got it